bookmarks/account/utils.py

```diff
@@ -14,20 +16,31 @@ def add_user(form):
     new_user.set_password(form.cleaned_data["password"])
     new_user.save()
     Profile.objects.create(user=new_user)
+    create_action(new_user, "has created an account")
     return new_user
 
 
-def follow_or_unfollow(user_id, from_user, action):
+def follow_or_unfollow_and_create_action(user_id, from_user, action):

```

Comment:
Where is this being used?

SRP violation. A method should do only one thing and do it well.

Even if we ignore SRP, it violates single level of abstraction.